pkg/workflow_execution.go

```diff
@@ -914,7 +914,7 @@ func (c *Client) FinishWorkflowExecutionStatisticViaExitHandler(namespace, name
 			"finished_at": time.Now().UTC(),
 			"phase":       phase,
 		}).
-		Where(sq.Eq{"name": name}).
+		Where(sq.Eq{"name": name}, sq.NotEq{"phase": "Terminated"}).
```

Comment:
@rushtehrani This needs to be changed to
```
		Where(sq.Eq{"name": name}).Where(sq.NotEq{"phase": "Terminated"}).
```
The 2nd parameter is ignored, and the "AND" piece is not added.
That's why it's not working.

Reply:
@aleksandrmelnikov good catch regarding `AND`, changing it to:
```go
               Where(sq.And{
			sq.Eq{"name": name},
			sq.NotEq{"phase": "Terminated"},
		}).
```

Reply:
@aleksandrmelnikov issue is resolved, you can test with this image: `onepanel/core:4fec204`

Reply:
Great, testing.